docs/reference/commandline/dockerd.md

```diff
@@ -700,6 +700,34 @@ ENOSPC and will shutdown filesystem.
 $ sudo dockerd --storage-opt dm.xfs_nospace_max_retries=0
 ```
 
+##### `dm.libdm_log_level`
+
+Specifies the maxmimum `libdm` log level that will be forwarded to `dockerd`'s
+log (as specified by `--log-level`). This option is primarily intended for
+debugging bugs involving interactions with `libdm`, and non-default values will
+likely cause erroneous warnings to be output when no actual error has occurred.
```

Comment:
Maybe "Using values other than the defaults will cause false positive warnings to be logged."